hack/functests.sh

```diff
@@ -22,6 +22,7 @@ set -e
 DOCKER_TAG=${DOCKER_TAG:-devel}
 DOCKER_TAG_ALT=${DOCKER_TAG_ALT:-devel_alt}
 KUBEVIRT_E2E_PARALLEL_NODES=${KUBEVIRT_E2E_PARALLEL_NODES:-3}
+KUBEVIRT_FUNC_TEST_GINKGO_ARGS=${FUNC_TEST_ARGS}
```

Comment:
Do you think it would make sense to change this line to

```
KUBEVIRT_FUNC_TEST_GINKGO_ARGS=${FUNC_TEST_ARGS:-${KUBEVIRT_FUNC_TEST_GINKGO_ARGS}}
```

Reply:
@rmohr Sure why not, we can make this var public as well. For the long term, the `FUNC_TEST_ARGS` will be permanently replaced with `KUBEVIRT_FUNC_TEST_GINKGO_ARGS`, for the sake of readability.